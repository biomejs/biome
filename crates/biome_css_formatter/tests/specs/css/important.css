html {
    font-size: 12px!important;
    font-size: 12px !important;
    color: red ! important;
    background-color: white !
important;
}