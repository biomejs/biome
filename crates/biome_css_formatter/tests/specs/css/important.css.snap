---
source: crates/biome_formatter_test/src/snapshot_builder.rs
info: css/important.css
---

# Input

```css
html {
    font-size: 12px!important;
    font-size: 12px !important;
    color: red ! important;
    background-color: white !
important;
}
```


=============================

# Outputs

## Output 1

-----
Indent style: Tab
Indent width: 2
Line ending: LF
Line width: 80
Quote style: Double Quotes
-----

```css
html {
	font-size: 12px !important;
	font-size: 12px !important;
	color: red !important;
	background-color: white !important;
}
```


